src/components/ErrorCard/index.jsx

```diff
@@ -13,18 +13,23 @@ const ErrorCard = ({
   content,
   buttonLabel,
   buttonHref,
-  buttonIcon
+  buttonIcon,
+  style
 }) => {
   return (
-    <div className={styles.ErrorCard}>
+    <div className={styles.ErrorCard} style={style}>
```

Comment:
This can be done when extracting it to cozy-ui, but I think we should spread props, so the component can receive events, custom classname, etc

Reply:
The ErrorCard will be removed in favor of using the Infos component from cozy-ui.

Will be done in https://github.com/cozy/cozy-banks/pull/1574